lib/v2/nasa/apod-cn.js

```diff
@@ -0,0 +1,53 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const rootUrl = `https://www.nasachina.cn/astronomy-picture-of-the-day`;
```

Comment:
This site has a JSON API. Please use that instead.
`/wp-json/wp/v2/categories` will list all the categories. The category id for apod is `2`.
`/wp-json/wp/v2/posts` will list all the posts. You can filter the posts by using `categories` search params and `per_page` if you want more than 10 posts.